tests/test_events_broker.py

```diff
@@ -0,0 +1,43 @@
+from unittest.mock import MagicMock
```

Comment:
According to our convention that should be used also in this repository, these tests should be put in:
```
tests/test_events/test_broker.py
```

Reply:
in LBZ we don't follow that convention yet.

Reply:
But we should and I asked for that in the previous pull requests 😉 

There won't be a better moment to start doing it better, especially since it requires ~1 minute of work.